AccessCheckoutDemo/AccessCheckoutDemo/CardFlowViewController.swift

```diff
@@ -130,11 +130,37 @@ class CardFlowViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
         panTextField.placeholder = "Card Number"
         expiryDateTextField.placeholder = "MM/YY"
         cvcTextField.placeholder = "CVC"
+        
+        //Apply onfocus listeners
+        panTextField.setOnFocusChangedListener{view, hasFocus in
+            if #available(iOS 13.0, *) {
+                view.borderColor = hasFocus ? .systemBlue : .systemGray5
+            } else {
+                view.borderColor = hasFocus ? .systemBlue : .systemGray
+            }
+        }
```

Comment:
Do you mind doing the same changes in the 2 other controllers so we keep some consistency across the demo applications UI?